Quick/Reporting/XCTestObservationCenter+Reporting.m

```diff
@@ -0,0 +1,135 @@
+//
+//  XCTestObservationCenter+Reporting.m
+//  Quick
+//
+//  Created by Paul Young on 1/2/15.
+//  Copyright (c) 2015 Brian Ivan Gesiak. All rights reserved.
+//
+
+#import "XCTestObservationCenter+Reporting.h"
+#import <XCTest/XCTestRun.h>
+#import "XCTestDriver.h"
+#import <objc/runtime.h>
+
+@implementation XCTestObservationCenter (Reporting)
+
++ (void)load {
+    NSDictionary *environment = [[NSProcessInfo processInfo] environment];
+    NSString *reporter = [environment objectForKey:@"QUICK_REPORTER"];
+    
+    if (!(reporter == nil || [reporter isEqualToString:@"xcode"])) {
```

Comment:
I tried various, apparently supported ways to do this but this was the only way I could get this to work so far:

```
QUICK_REPORTER=nyan xcodebuild -workspace Foo.xcworkspace -scheme Foo clean test
```

`-quickReporter=nyan` accessed via `NSUserDefaults` would be a lot nicer.